src/main/cpp/chassis/commands/DriveToAprilTagTarget.cpp

```diff
@@ -0,0 +1,121 @@
+//====================================================================================================================================================
+// Copyright 2025 Lake Orion Robotics FIRST Team 302
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
```

Comment:
Since I don't see vision stuff in here, are you seeing a vision drivetoapriltag type command and then a sequential command to tie this command (odometry-based drive to apriltag and vision drive to apriltag) together?

Reply:
Correct, this is basically drive to a target based on odometery and AprilTag poses with field offsets. We can do actually vision based correction in the DriveToVisionTarget and use commands and zones to trigger the switch